StudioCore/ParamEditor/MassParamEdit.cs

```diff
@@ -647,16 +647,46 @@ private void Setup()
                 return val.GetType() == typeof(byte[]) ? ParamUtils.Dummy8Write((byte[])val) : val.ToString();
             }));
             argumentGetters.Add("field", (1, (field) => (param) => {
-                PseudoColumn pc = field[1].Equals("ID") ? PseudoColumn.ID : field[1].Equals("Name") ? PseudoColumn.Name : PseudoColumn.None;
-                Param.Column? col = param?[field[1]];
+                PseudoColumn pc = field[0].Equals("ID") ? PseudoColumn.ID : field[0].Equals("Name") ? PseudoColumn.Name : PseudoColumn.None;
+                Param.Column? col = param?[field[0]];
                 if (pc == PseudoColumn.None && col == null)
-                    throw new Exception($@"Could not locate field {field[1]}");
+                    throw new Exception($@"Could not locate field {field[0]}");
                 return (row) => {
                     object val = pc == PseudoColumn.ID ? row.ID : pc == PseudoColumn.Name ? row.Name : row[col].Value;
                     string v = val.GetType() == typeof(byte[]) ? ParamUtils.Dummy8Write((byte[])val) : val.ToString();
                     return (c) => v;
                 };
             }));
+            argumentGetters.Add("average", (2, (field) => (param) => {
+                PseudoColumn pc = field[0].Equals("ID") ? PseudoColumn.ID : field[0].Equals("Name") ? PseudoColumn.Name : PseudoColumn.None;
```

Comment:
I feel like this copy paste code could be refactored into an additional helper function that takes a function to calculate the statistic?

Reply:
I'd say let's try to use the DRY (don't repeat yourself) principle and refactor this code more to reduce the copy paste and make it easier to add more statistics.